docs/introduction/media/media.md

```diff
@@ -0,0 +1,16 @@
+# Media
+
+# Blogs
+- [M3: Uber’s Open Source, Large-scale Metrics Platform for Prometheus](https://eng.uber.com/m3/) By Rob Skillington - Aug 7, 2018.
+
+# Recorded Talks
+- [Panel Discussion: Prometheus Long-Term Storage Approaches](https://youtu.be/VvJx0WTiGcA?t=23530) including highlights of M3 stack by Nikunj Agarwal - Aug 9, 2018.
+
+- [Putting billions of timeseries to work at Uber with autonomous monitoring](https://vimeo.com/274821002) By Prateek Rungta - Jun 6, 2018. [Slides](http://bit.ly/m3db-monitorama2018)
+
+# Upcoming Talks
+- [Keynote: Smooth Operator♪: Large Scale Automated Storage with Kubernetes](https://kccna18.sched.com/event/Gsxn/keynote-smooth-operator-large-scale-automated-storage-with-kubernetes-celina-ward-software-engineer-matt-schallert-site-reliability-engineer-uber) By Celina Ward & Matt Schallert - Dec 13, 2018.
+
+- [Learnings, patterns and Uber’s metrics platform M3, open sourced as a Prometheus long term storage backend](https://osmc.de/events/en-learnings-patterns-and-ubers-metrics-platform-m3-open-sourced-as-a-prometheus-long-term-storage-backend/) By Rob Skillington - Nov 5, 2018.
```

Comment:
Are these here because there are no recordings of it? This looks like it's part of the `Upcoming Talks` section and they're in the past.

Reply:
pretty much. we'll update as and when the videos get uploaded.